content/php/concepts/string-functions/terms/echo/echo.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: 'echo()'
+Description: 'Outputs one or more strings.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Output'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/web-development'
+---
+
+> The metadata above provides information about the entry for the PHP String Functions echo().
+
```

Comment:
```suggestion
```
Remove extra line break

Reply:
Hello! I have made the requested changes to address the feedback provided.  Thank you!